ui/src/components/query/builder/display/EditDisplayMatch.vue

```diff
@@ -23,7 +23,8 @@
       />
     </div>
     <div v-else-if="match.description" v-html="match.description" @dblclick="editMatch()"></div>
-    <div v-if="match.nodeRef" class="node-ref" v-html="getDisplayFromNodeRef(match.nodeRef)"></div>
+    <div v-if="match.nodeRef" class="node-ref" v-html="getDisplayFromNodeRef(match.nodeRef)" @dblclick="editMatch()"></div>
```

Comment:
does that work for a dbclick on a property inside a match?

Reply:
yes 👍